actions/campaigns.go

```diff
@@ -453,3 +453,40 @@ func GetCampaignClicksStats(c *gin.Context) {
 		ClicksStats: stats,
 	})
 }
+
+func GetCampaignComplaintsStats(c *gin.Context) {
```

Comment:
You can omit the Stats from the name, GetCampaignComplaints is more appropriate.